### PR TITLE
Addressing Server Admin review comments

### LINKED ISSUE #24643: Address comments on the Server Adminstration guide
### Description

Miscellaneous comments

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #24643

@ahus1 @vaceksimon Can you verify this.   This addressed most issues you raised.

### CODE REVIEW

**Review by ahus1 — DISMISSED** [review]
@andymunro - your PR references a GitHub issue which is not related to the PR. GitHub PRs can only reference GitHub issues, not internal JIRA issues. Please duplicate the JIRA issue to a public GitHub issue first so it is clear what you intent to change. 

UPDATE: the commit in this PR doesn't contain a signoff, please add this as well. See https://www.keycloak.org/2023/10/dco for the details.

File: docs/documentation/server_admin/topics/authentication/flows.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/flows.adoc:126]: I apologize for the incorrect comment I left on the gear icon on the Jira ticket. The gear menu item to configure the execution is still there, it's just not there for every single execution.

File: docs/documentation/server_admin/topics/authentication/flows.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/flows.adoc:264]:
```suggestion
. Click *⚙️* (gear icon) menu of the *Conditional - Level Of Authentication*
```

File: docs/documentation/server_admin/topics/admin-cli.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1936]:
```suggestion
$ kcadm.sh get "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```

File: docs/documentation/server_admin/topics/admin-cli.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1951]:
```suggestion
$ kcadm.sh update "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm -b '{"id":"dd91611a-d25c-421a-87e2-227c18421833","alias":"my_otp_config","config":{"x509-cert-auth.extendedkeyusage":"","x509-cert-auth.mapper-selection.user-attribute-name":"usercertificate","x509-cert-auth.ocsp-responder-uri":"","x509-cert-auth.regular-expression":"(.*?)(?:$)","x509-cert-auth.crl-checking-enabled":"true","x509-cert-auth.confirmation-page-disallowed":"","x509-cert-auth.keyusage":"","x509-cert-auth.mapper-selection":"Custom Attribute Mapper","x509-cert-auth.crl-relative-path":"crl.pem","x509-cert-auth.crldp-checking-enabled":"false","x509-cert-auth.mapping-source-selection":"Match SubjectDN using regular expression","x509-cert-auth.ocsp-checking-enabled":""}}'
```

File: docs/documentation/server_admin/topics/admin-cli.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/admin-cli.adoc:1966]:
```suggestion
$ kcadm.sh delete "authentication/config/dd91611a-d25c-421a-87e2-227c18421833" -r demorealm
```

File: docs/documentation/server_admin/topics/authentication/otp-policies.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/otp-policies.adoc:9]:
```suggestion
. Click the *Policies* tab.
```

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/webauthn.adoc:77]:
```suggestion
```

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/authentication/webauthn.adoc:80]: This step is perfectly fine and should stay. Step 2 on line 77 is supposed to be the one deleted

File: docs/documentation/server_admin/topics/users/proc-requiring-tac-agreement-at-registration.adoc
Comment (vaceksimon) [inline docs/documentation/server_admin/topics/users/proc-requiring-tac-agreement-at-registration.adoc:20]:
```suggestion
. Click the *Flows* tab.
```

File: docs/documentation/server_admin/images/user-profile-select-options-simple-i18n.png
Comment (vaceksimon) [inline docs/documentation/server_admin/images/user-profile-select-options-simple-i18n.png:1]: The picture from an older version of Keycloak I had in mind was [user-profile-select-options-custom-validator.png](https://github.com/keycloak/keycloak/blob/main/docs/documentation/server_admin/images/user-profile-select-options-custom-validator.png). This picture is fine as it is.

**Review by vaceksimon — CHANGES_REQUESTED** [review]
After these corrections it's ready for merge. Good job Andy

**Comment by andymunro** [comment]
@ahus1 @vaceksimon Comments addressed and signed off to correct Jira.

**Review by ahus1 — DISMISSED** [review]
DCO looks good to me. Waiting for review of @vaceksimon

**Review by vaceksimon — APPROVED** [review]
LGTM! Ready to merge @ahus1

**Comment by andymunro** [comment]
@ahus1 I can fix one more issue with this guide related to the Account Console broken link

**Comment by andymunro** [comment]
@ahus1 I fixed that last broken link issue.  Ready to merge.

**Review by ahus1 — APPROVED** [review]
LGTM

**Pull request merged**
